### PR TITLE
Disable flaky test TestQuantLoweringCustomBackendPass

### PR DESCRIPTION
**Pull request by guangy10**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #2465

Differential Revision: [D54966578](https://our.internmc.facebook.com/intern/diff/D54966578)

### CODE REVIEW

**Comment by guangy10** [comment]
@guangy10 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D54966578).

File: exir/tests/test_quant_lowering_custom_backend_pass.py [hunk 1]
```diff
@@ -506,6 +507,7 @@ class TestQuantLoweringCustomBackendPass(unittest.TestCase):
     def setUp(self) -> None:
         super().setUp()
 
+    @unittest.skipIf(IS_LINUX, "Flaky test on Linux x86, needs debugging")
```
Comment (huydhn) [inline exir/tests/test_quant_lowering_custom_backend_pass.py:510]: May be create an issue to keep track of the issue and link it here, but it's up to you

**Review by huydhn — APPROVED** [review]

File: exir/tests/test_quant_lowering_custom_backend_pass.py (hunk 1 above)
Comment (huydhn) [inline exir/tests/test_quant_lowering_custom_backend_pass.py:510]: Also, if the test is failing consistently (not flaky), a nicer option would be to mark it as XFAIL.  For flaky test, this is the right choice

Comment (guangy10) [reply exir/tests/test_quant_lowering_custom_backend_pass.py]: Linked to the GitHub issue

**Comment by guangy10** [comment]
@guangy10 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D54966578).

**Pull request closed without merging**